ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -113,19 +100,19 @@ void cleanUp() {
         if (mVungleBannerAd != null) {
             Log.d(TAG, "Vungle banner adapter cleanUp: destroyAd # " + mVungleBannerAd.hashCode());
             mVungleBannerAd.destroyAd();
-            if (mVungleBannerAd.getParent() != null) {
-                Log.d(TAG, "Vungle banner adapter cleanUp: removeView");
+            if (mVungleBannerAd != null && mVungleBannerAd.getParent() != null) {
```

Comment:
We should double check null value here in banner leak case.
1.We receive playback error when fire destroyAd() if banner not display(not attachedToWindow).
2.We don't receive playback error when fire destroyAd if banner displayed(attachedToWindow).

in 1 case: its banner leak caused by adapter not receive onDestroy and not remove map timely.

Reply:
@HuFangshuai  how it can be null with mVungleBannerAd != null check above?

Reply:
Condition: banner leak exists in map and same ad request loading.
debug info: loadBanner->createBanner->cleanUp-> mVungleBannerAd#destroyAd->mAdPlayCallback#onError(OPERATION_CANCELED)->manager#removeActiveBannerAd->cleanUp->removeView->set null value->removeView->NPE